interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -693,6 +693,8 @@ namespace cling {
     clang::Sema& getSema() const;
     clang::DiagnosticsEngine& getDiagnostics() const;
 
+    inline IncrementalCUDADeviceCompiler* getCUDACompiler() const;
```

Comment:
```suggestion
    IncrementalCUDADeviceCompiler* getCUDACompiler() const {
      return m_CUDACompiler.get();
    }
```

Reply:
> That `inline` won't work, I meant to suggest to _actually_ inline :-) See proposed changes.

Oh, I misunderstood you :sweat_smile: I changed the code and inlined the getter in the header file.